### DISCUSSION TITLE
BAG ?

### DISCUSSION THREAD

**Discussion by Canek7793** [MAFINS/MenyooSP#525] [General]
Changing the language in the trainer does not work, when trying to change the language, it gives an error with ScriptHookV, despite the fact that the latest version of ScriptHookV and the trainer is used.

**Comment by BSK-YouTube** [comment]
The problem manifests itself - when we load "ScriptHookVDotNet" into the folder with the game, without "ScriptHookVDotNet" - everything works fine, nevertheless it is clearly some kind of incompatibility and it would not be bad to solve this issue!

**Comment by BSK-YouTube** [comment]
Dear Mafins, **please pay attention to this problem**.
